source/GameInterface/CompatibilityInfo.cs

```diff
@@ -122,35 +123,41 @@ public override bool Equals(object obj)
             return false;
         }
 
-        public static CompatibilityInfo Deserialize(byte[] serialized)
+        private CompatibilityInfo()
         {
-            var serializedString = System.Text.Encoding.ASCII.GetString(serialized);
-            var stringReader = new System.IO.StringReader(serializedString);
-            var serializer = new XmlSerializer(typeof(CompatibilityInfo));
-
-            return (CompatibilityInfo)serializer.Deserialize(stringReader);
+            Modules = new List<ModuleInfo>();
```

Comment:
Move this initialization to [line 73](https://github.com/Bannerlord-Coop-Team/BannerlordCoop/pull/461/files)